sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -40,6 +41,12 @@ def __init__(self, shell, data=None, spark_events=None):
     def execute_spark(self, cell, output_var, samplemethod, maxrows, samplefraction, session_name, coerce):
         (success, out) = self.spark_controller.run_command(Command(cell), session_name)
         if not success:
+            if conf.spark_statement_errors_are_fatal():
```

Comment:
I'm not sure how the repo is treating errors but I would imagine you'd always want to raise here and let the user handle exceptions if they wishes. In the interactive case later evaluation cells could still reinteract with the session just fine.

Reply:
I agree, but I was worried about changing the current default functionality, I 100% agree with you though.